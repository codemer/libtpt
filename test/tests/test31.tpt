@set(array, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)\
@foreach (-2, -1, 0, array, 11) {
	@<@if (${.} == 3) { @next}
	@elsif (${.} == 7) { @last}
	@<${.} \
}

