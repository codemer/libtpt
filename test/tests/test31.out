-2 -1 0 1 2 4 5 6 
